flink-state-backends/flink-statebackend-forst/src/main/java/org/apache/flink/state/forst/fs/ForStFlinkFileSystem.java

```diff
@@ -340,8 +373,9 @@ public boolean delete(Path path, boolean recursive) throws IOException {
         if (localPathTuple.f0) {
             success = localFS.delete(localPathTuple.f1, recursive); // delete from local
         }
-        success |= delegateFS.delete(path, recursive); // and delete from remote
+        success |= fileMappingManager.deleteFile(path, recursive);
```

Comment:
I'm wondering if the `create` won't affect  the fileMapping, then how `fileMappingManager.deleteFile` delete the created but not mapped files?

Reply:
Currently, files built by `create` won't be put into `fileMapping` hash map.
 `fileMappingManager.deleteFile` would delete the files not in `filemapping` directly.
 
 I will change the description of `FileMappingManager`.